config/_default/menus/menus.en.yaml

```diff
@@ -2874,31 +2874,36 @@ main:
     parent: rum
     identifier: rum_ios
     weight: 3
+  - name: SwiftUI
+    url: real_user_monitoring/ios/swiftui/
+    parent: rum_ios
+    identifier: rum_ios_swiftui
+    weight: 301
```

Comment:
I think we can move this down the `Advanced Configuration` section, makes sense to have this close to `Mobile Vitals` and `Web View Tracking`